src/components/Audiences/processDestinationsFactory.js

```diff
@@ -34,7 +34,7 @@ const processUrls = (fireReferrerHideableImage, logger, destinations) => {
           logger.error(createResultLogMessage(urlDestination, false));
         });
     })
-  );
+  ).then(noop);
```

Comment:
@ninaceban Would you please update the tests accordingly to assert that we're not resolving the promise with a value?

Reply:
I'll add tests. thank you